code/game/jobs/job/support_chaplain.dm

```diff
@@ -33,8 +33,7 @@
 	if(visualsOnly)
 		return
 
-	if(H.mind)
-		H.mind.isholy = TRUE
+	ADD_TRAIT(H.mind, TRAIT_HOLY, ROUNDSTART_TRAIT)
```

Comment:
Does this automatically nullcheck? I can't check myself because I am unable to go near the repo for a while

Reply:
it does not